common/config/rush/pnpm-config.json

```diff
@@ -1,6 +1,8 @@
 {
     "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/pnpm-config.schema.json",
     "globalOverrides": {
-      "caniuse-lite": "1.0.30001481"
+      "caniuse-lite": "1.0.30001481",
+      "postcss": "8.4.20",
+      "autoprefixer": "10.4.14"
```

Comment:
Your comment says iTwinUI has a dependency on autoprefixer.. It does not. Why this was needed...?

Reply:
It is locked in itwinUI repo because they are also using storybook 6.x
![image](https://github.com/iTwin/admin-components-react/assets/64415995/a0e3810f-703b-46c9-b5fc-51b5bb3bcf8c)

https://github.com/iTwin/iTwinUI/blob/dev/package.json

and without it we get errors from itwinui-css package, I also raised this issue in itwinui teams channel and they recommended to update autoprefixer

Reply:
oh, so it is storybook issue then.. ok.